pulser/tests/test_simresults.py

```diff
@@ -97,6 +102,26 @@ def test_get_final_state():
                              np.abs(results.states[-1].full()), atol=1e-5))
 
 
+def test_get_final_state_noisy():
+    seq_ = Sequence(reg, Chadoq2)
+    seq_.declare_channel('ram', 'raman_local', initial_target="A")
+    seq_.add(pi, 'ram')
+    noisy_config = SimConfig(noise=('SPAM', 'doppler'))
+    sim_noisy = Simulation(seq_, config=noisy_config)
+    res3 = sim_noisy.run()
+    res3._meas_basis = 'digital'
+    res3.get_final_state()
+    res3._meas_basis = 'ground-rydberg'
+    res3.get_final_state()
+    res3.states
+    res3.results
```

Comment:
The objective of the tests is not just to make sure every line is executed, but to assert that the outcomes are what you expect. Add some meaningful `assert` statements.

Reply:
I will. The thing with noise effects is the results are random and it's hard to come up with meaningful tests.